src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -1850,99 +1833,56 @@ export default Vue.extend({
               this.player.exitPictureInPicture()
             }
             break
-          case 49:
-            // 1 Key
-            // Jump to 10% in the video
-            event.preventDefault()
-            this.changeDurationByPercentage(0.1)
-            break
-          case 50:
-            // 2 Key
-            // Jump to 20% in the video
-            event.preventDefault()
-            this.changeDurationByPercentage(0.2)
-            break
-          case 51:
-            // 3 Key
-            // Jump to 30% in the video
-            event.preventDefault()
-            this.changeDurationByPercentage(0.3)
-            break
-          case 52:
-            // 4 Key
-            // Jump to 40% in the video
+          case '0':
```

Comment:
I guess if we allow customizing keybinds in the future then we can exclude using different shortcuts for this